reference-architecture/aws-ansible/ansible.cfg

```diff
@@ -0,0 +1,22 @@
+# config file for ansible -- http://ansible.com/
+# ==============================================
+[defaults]
+#callback_plugins = ../openshift-ansible/ansible-profile/callback_plugins
+forks = 50
+host_key_checking = False
+hostfile = inventory/aws/hosts/ec2.py
+remote_user = ec2-user
+gathering = smart
+fact_caching = jsonfile
+fact_caching_connection = .ansible/cached_facts
+fact_caching_timeout = 900
```

Comment:
I'm good with removing that.

Ryan

On Aug 30, 2016 11:13 PM, "Jason DeTiberus" <notifications@github.com>
wrote:

> In reference-architecture/aws-ansible/ansible.cfg
> <https://github.com/openshift/openshift-ansible-contrib/pull/13#discussion_r76918075>
> :
>
> > @@ -0,0 +1,22 @@
> > +# config file for ansible -- http://ansible.com/
> > +# ==============================================
> > +[defaults]
> > +#callback_plugins = ../openshift-ansible/ansible-profile/callback_plugins
> > +forks = 50
> > +host_key_checking = False
> > +hostfile = inventory/aws/hosts/ec2.py
> > +remote_user = ec2-user
> > +gathering = smart
> > +fact_caching = jsonfile
> > +fact_caching_connection = .ansible/cached_facts
> > +fact_caching_timeout = 900
>
> I would suggest against using fact caching, unless it will only be used
> through the wrapper script and the wrapper clears the fact cache between
> runs.
>
> Only reason I say that is because the fact caching will cause unintuitive
> behavior for users that aren't familiar with the feature in advance.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible-contrib/pull/13/files/44578f7931ece34bed51d9f58b0f81c9bab33b02#r76918075>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADoslPmr1N0CasSUTOX2q8bFwX4QG7e0ks5qlPFXgaJpZM4JxJba>
> .
>